lib/miq_automation_engine/service_models/miq_ae_service_floating_ip.rb

```diff
@@ -4,5 +4,8 @@ class MiqAeServiceFloatingIp < MiqAeServiceModelBase
     expose :vm,                    :association => true
     expose :cloud_tenant,          :association => true
     expose :network_port,          :association => true
+
+    expose :raw_update_floating_ip
```

Comment:
Ok, this was inspired by existing code implementing the wait_for_task queuing.
My understanding is they are stub for future control. Now I'd love to have the big picture of this queue approach so I don't do 1 steps forward, 2 backwards. Anyway, in the meantime, I remove the studs.